backend/Dockerfile

```diff
@@ -2,16 +2,18 @@ FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
 WORKDIR /app
 EXPOSE 5000
 
-FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+FROM rust:1.65 AS rust-builder
 WORKDIR /src
-COPY ["Application/Application.csproj", "Application/"]
-RUN dotnet restore "Application/Application.csproj" 
```

Comment:
Discussed offline, included here for reference: Calling `dotnet restore` before `COPY` improves chances of a cache hit from the docker daemon.

Doesn't matter for our build because we never reuse build VMs and don't have a distributed image cache, but is generally good practice.